app/v1/models/requestData/CreateBFLossRawData.scala

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
Either this should be named `CreateBroughtForwardLossRawData`, or the `CreateBroughtForwardLossResponse` model should be named `CreateBFLossResponse` (to keep it consistent).

Reply:
good point i'll change it